models.py

```diff
@@ -11,13 +11,13 @@ class HadithCollection(db.Model):
     def serialize(self):
         return {
             'name': self.name,
-            'has_books': self.hasbooks,
+            'short_intro': self.shortintro,
+            'has_books': self.hasbooks == "yes",
+            'has_chapters': self.haschapters == "yes",
             'title_en': self.englishTitle,
             'title_ar': self.arabicTitle,
-            'num_hadith': self.numhadith,
             'total_hadith': self.totalhadith,
-            'has_chapters': self.haschapters,
-            'short_intro': self.shortintro,
+            'numhadith': self.numhadith,
```

Comment:
Let's keep the underscore (especially since `total_hadith` has an underscore). And maybe call it `num_available_hadith`. And we can explain the difference in the documentation.

Reply:
Got it, I intended to do so. But somehow missed. :+1: